integr8ly/integreatly-operator
# MGDAPI-4721 update OO to 4.0.0

# Issue link
https://issues.redhat.com/browse/MGDAPI-4721

# What
Updated OO to official 4.0.0 image

# Verification steps
- Install 1.29.0 from index (you can use quay.io/mstoklus/managed-api-service-index:1.29.0)
- Create index for 1.30.0 using this branch
- Upgrade from 1.29.0 to 1.30.0
- Verify upgrade was successful
- Verify all required e2e tests passed on this PR

- navigate to customer monitoring grafana and confirm that the rate limiting dashboard works as expected.
